src/gamedata/resourcefiles/file_wad.cpp

```diff
@@ -38,7 +38,6 @@
 #include "w_wad.h"
 #include "gi.h"
 #include "doomerrors.h"
-
```

Comment:
Was it an intentional change?

Reply:
No, that wasn't. The file was changed in the process of trying to fix the compile, but spaces are hard to remember when healing the file back after changing it.